src/Core/Commander/Providers/Fetcher.js

```diff
@@ -2,6 +2,8 @@ import { TextureLoader } from 'three';
 
 const textureLoader = new TextureLoader();
 
+textureLoader.setCrossOrigin('');
```

Comment:
This is https://html.spec.whatwg.org/multipage/embedded-content.html#attr-img-crossorigin (`TextureLoader` delegates to `ImageLoader` which simply creates a `new Image()`)

Based on that, `setCrossOrigin('')` is equivalent to `setCrossOrigin('anonymous')`, and only `''`, `'anonymous'` and `'use-credentials'` are valid values.

The real change in r84 is that `ImageLoader` used to delegate to a `FileLoader`/`XHRLoader` (the `new Image` path in r83 was only triggered [when `crossOrigin` was not `undefined`](https://github.com/mrdoob/three.js/commit/73a4ca56b51dcd9480bcbf62b22c904dcb6c8ffb), or when loading a `data:` URL; prior to r83, only `data:` URLs triggered the `new Image` path, everything else going through the `XHRLoader`), but that was removed in r84: https://github.com/mrdoob/three.js/commit/e7f7d330b975b1e7d3b760a3b943976406bfa814

So, cross-origin images used to work until r84 because they were loaded as blobs _via_ an XHR, but they're now loaded "as images", so they respect the canvas security policy wrt cross-origin media.